docs/execution_nodes.md

```diff
@@ -0,0 +1,78 @@
+# Adding execution nodes to AWX
+
+Stand-alone execution nodes can be added to run alongside the Kubernetes deployment of AWX. These machines will not be a part of the AWX Kubernetes cluster. The control nodes running in the cluster will connect and submit work to these machines via Receptor. The machines be registered in AWX as type "execution" instances, meaning they will only be used to run AWX Jobs (i.e. they will not dispatch work or handle web requests as control nodes do).
+
+Below is an example of a single AWX pod connecting to two different execution nodes. For each execution node, the awx-ee container makes an outbound TCP connection to the machine via Receptor.
+
+```
+                                 AWX POD
+                             ┌──────────────┐
+                             │              │
+                             │ ┌──────────┐ │
+┌─────────────────┐          │ │ awx-task │ │
+│ execution node 1│◄────┐    │ ├──────────┤ │
+├─────────────────┤     ├────┼─┤ awx-ee   │ │
+│ execution node 2│◄────┘    │ ├──────────┤ │
+└─────────────────┘ Receptor │ │ awx-web  │ │
+                      TCP    │ └──────────┘ │
+                     Peers   │              │
+                             └──────────────┘
+```
+
+Note, if the AWX deployment is scaled up, the new AWX pod will also make TCP connections to each execution node.
+
+
+## Overview
+Adding an execution instance involves a handful of steps:
+
+1. [Start a machine that is accessible from the k8s cluster (Red Hat family of operating systems are supported)](#start-machine)
+2. [Create a new AWX Instance with `hostname` being the IP or DNS name of your remote machine.](#create-instance-in-awx)
+3. [Download the install bundle for this newly created instance.](#download-the-install-bundle)
+4. [Run the install bundle playbook against your remote machine.](#run-the-install-bundle-playbook)
+5. [Wait for the instance to report a Ready state. Now jobs can run on that instance.](#wait-for-instance-to-be-ready)
+
+
+### Start machine
+
+Bring a machine online with a compatible Red Hat family OS (e.g. RHEL 8 and 9). This machines needs a static IP, or a resolvable DNS hostname that the AWX cluster can access. The machine will also need an available open port to establish inbound TCP connections on (default is 27199).
+
+In general the more CPU cores and memory the machine has, the more jobs that can be scheduled to run on that machine at once. See https://docs.ansible.com/automation-controller/4.2.1/html/userguide/jobs.html#at-capacity-determination-and-job-impact for more information on capacity.
+
+
+### Create Instance in AWX
```

Comment:
You aren't being entirely consistent about the capitalization of Instance in these headlines.  This one is, two more below are not.  Just a picky thing.